cognite/neat/core/_client/_api/statistics.py

```diff
@@ -0,0 +1,91 @@
+import itertools
+from typing import overload
+
+from cognite.client._api_client import APIClient
+from cognite.client._cognite_client import CogniteClient
+from cognite.client.config import ClientConfig
+from cognite.client.data_classes.data_modeling.ids import _load_space_identifier
+from cognite.client.utils.useful_types import SequenceNotStr
+
+from cognite.neat.core._client.data_classes.statistics import (
+    ProjectStatsAndLimits,
+    SpaceInstanceCounts,
+    SpaceInstanceCountsList,
+)
+
+
+class StatisticsAPI(APIClient):
+    _RESOURCE_PATH = "/models/statistics"
+
+    def __init__(self, config: ClientConfig, api_version: str | None, cognite_client: CogniteClient) -> None:
+        # This is an alpha API, which requires a specific version.
+        super().__init__(config, "v1", cognite_client)
+        self._RETRIEVE_LIMIT = 100
+
+    def project(self) -> ProjectStatsAndLimits:
+        """`Retrieve project-wide usage data and limits
+
+        Returns the usage data and limits for a project's data modelling usage, including data model schemas
+        and graph instances
+
+        Returns:
+            ProjectStatsAndLimits: The requested statistics and limits
+
+        Examples:
+            Fetch project statistics (and limits) and check the current number of data models vs.
+            and how many more can be created:
+                >>> from cognite.client import CogniteClient
+                >>> client = CogniteClient()
+                >>> stats = client.data_modeling.statistics.project()
+                >>> num_dm = stats.data_models.current
+                >>> num_dm_left = stats.data_models.limit - num_dm
```

Comment:
Are these examples accurate? The relation to the written code isn't clear to me.

Reply:
Yes, the `ProjectStatsAndLimits` object there is a `data_models` attribute which has `.current` and `.limit`.

Reply:
I mean, it's not clear how it relates to the `StatisticsAPI` class that it's an example for? Presumably it should somehow be using that?